### PR TITLE
Some RTL fix

### PR DESCRIPTION
**Pull request by Arseny271**

### CODE REVIEW

**Review by Maxxum69 — COMMENTED** [review]
Leave a review

**Review by ikyma — COMMENTED** [review]
Don't approve

File: app/src/main/java/org/thunderdog/challegram/util/text/Text.java [hunk 1]
```diff
@@ -108,7 +111,6 @@ public class Text implements Runnable, Emoji.CountLimiter, CounterAnimator.TextD
   private static final int FLAG_ABORT_PROCESS = 1 << 25;
   private static final int FLAG_FAKE_BOLD = 1 << 26;
   private static final int FLAG_FULL_RTL = 1 << 27;
-  private static final int FLAG_MAY_APPLY_RTL = 1 << 28;
```
Comment (msampley24) [inline app/src/main/java/org/thunderdog/challegram/util/text/Text.java:111]: 112

**Review by taha25485 — COMMENTED** [review]
💤

**Review by Kitwana150 — COMMENTED** [review]
Look show
IMG-20240421-WA0022

**Review by ahmetwrl01 — COMMENTED** [review]
- [ ]

**Review by ahmetwrl01 — COMMENTED** [review]
Dd

**Review by Johnnynzcryptonite — COMMENTED** [review]
app/src/main/java/org/thunderdog/challegram/util/text/Text.java

**Review by vkryl — APPROVED** [review]

**Comment by cumcum8197** [comment]
89edd75770858caa03eab66ba3d9adb5ba834082

**Pull request merged**